src/main.tsx

```diff
@@ -1,10 +1,23 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
-import App from "./pages/App.tsx";
+import { Provider } from "react-redux/es/exports";
+import store from "./modules/core/store";
+
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import { ROUTER } from './modules/core/constants/navigation';
 
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <React.StrictMode>
-    <App />
+    <Provider store={store}>
+      <BrowserRouter>
+        <Routes>
+          {ROUTER.map(({ path, Element }) => (
```

Comment:
It's better to avoid mapping in this part since dynamic generation of the routing part involves additional performance, which is not required at this stage. But you can leave it as is.